sequencer/src/bin/bridge.rs

```diff
@@ -0,0 +1,343 @@
+use anyhow::{bail, ensure, Context};
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use async_std::{sync::Arc, task::sleep};
+use clap::Parser;
+use contract_bindings::fee_contract::FeeContract;
+use es_version::SequencerVersion;
+use ethers::{
+    middleware::{Middleware, SignerMiddleware},
+    providers::Provider,
+    types::{Address, BlockId, U256},
+};
+use futures::stream::StreamExt;
+use jf_merkle_tree::{
+    prelude::{MerkleProof, Sha3Node},
+    MerkleTreeScheme,
+};
+use sequencer::{
+    eth_signature_key::EthKeyPair,
+    state::{FeeAccount, FeeAmount, FeeMerkleTree},
+    Header,
+};
+use std::time::Duration;
+use surf_disco::{error::ClientError, Url};
+
+type EspressoClient = surf_disco::Client<ClientError, SequencerVersion>;
+
+type FeeMerkleProof = MerkleProof<FeeAmount, FeeAccount, Sha3Node, { FeeMerkleTree::ARITY }>;
+
+/// Command-line utility for working with the Espresso bridge.
+#[derive(Debug, Parser)]
+enum Command {
+    Deposit(Deposit),
+    Balance(Balance),
+    L1Balance(L1Balance),
+}
+
+/// Deposit ETH from the L1 into Espresso.
+#[derive(Debug, Parser)]
+struct Deposit {
+    /// L1 JSON-RPC provider.
+    #[clap(short, long, env = "L1_PROVIDER")]
+    rpc_url: Url,
+
+    /// Espresso query service provider.
+    ///
+    /// This must point to an Espresso node running the node, availability, and Merklized state
```

Comment:
I think the second use of "node" here is ambiguous.

Reply:
Maybe I can put `/node` to make it clearer?

Reply:
See [2ac26c0](https://github.com/EspressoSystems/espresso-sequencer/pull/1390/commits/2ac26c031bb6c55f818660fe482c8ecbc8232093)

Reply:
Maybe. Thinking about it... it isn't really ambiguous, b/c it is referring to the `NodeApi`. But it still feels confusing b/c "node" is such a generic word. I suggest we capitalize "Node" and "Availability", but also it is a *nit* so not worth blocking over.